doc/api/os.md

```diff
@@ -33,205 +44,294 @@ defined are described in [OS Constants][].
+value is an array of objects that each describe an assigned network address.
+
+The properties available on the assigned network address object include:
+
+* `address` {String} The assigned IPv4 or IPv6 address
+* `netmask` {String} The IPv4 or IPv6 network mask
+* `family` {String} Either `IPv4` or `IPv6`
+* `mac` {String} The MAC address of the network interface
+* `internal` {boolean} `true` if the network interface is a loopback or
+  similar interface that is not remotely accessible; otherwise `false`
+* `scopeid` {number} The numeric IPv6 scope ID (only specified when `family`
+  is `IPv6`)
 
 ```js
-{ lo:
-   [ { address: '127.0.0.1',
-       netmask: '255.0.0.0',
-       family: 'IPv4',
-       mac: '00:00:00:00:00:00',
-       internal: true },
-     { address: '::1',
-       netmask: 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff',
-       family: 'IPv6',
-       mac: '00:00:00:00:00:00',
-       internal: true } ],
-  eth0:
-   [ { address: '192.168.1.108',
-       netmask: '255.255.255.0',
-       family: 'IPv4',
-       mac: '01:02:03:0a:0b:0c',
-       internal: false },
-     { address: 'fe80::a00:27ff:fe4e:66a1',
-       netmask: 'ffff:ffff:ffff:ffff::',
-       family: 'IPv6',
-       mac: '01:02:03:0a:0b:0c',
-       internal: false } ] }
+{
+  lo: [
+    {
+      address: '127.0.0.1',
+      netmask: '255.0.0.0',
+      family: 'IPv4',
+      mac: '00:00:00:00:00:00',
+      internal: true
+    },
+    {
+      address: '::1',
+      netmask: 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff',
+      family: 'IPv6',
+      mac: '00:00:00:00:00:00',
+      internal: true
+    }
+  ],
+  eth0: [
+    {
+      address: '192.168.1.108',
+      netmask: '255.255.255.0',
+      family: 'IPv4',
+      mac: '01:02:03:0a:0b:0c',
+      internal: false
+    },
+    {
+      address: 'fe80::a00:27ff:fe4e:66a1',
+      netmask: 'ffff:ffff:ffff:ffff::',
+      family: 'IPv6',
+      mac: '01:02:03:0a:0b:0c',
+      internal: false
+    }
+  ]
+}
 ```
 
-Note that due to the underlying implementation this will only return network
-interfaces that have been assigned an address.
-
 ## os.platform()
 <!-- YAML
 added: v0.5.0
 -->
 
-Returns the operating system platform. Possible values are `'darwin'`,
-`'freebsd'`, `'linux'`, `'sunos'` or `'win32'`. Returns the value of
-[`process.platform`][].
+The `os.platform()` method returns a string identifying the operating system
+platform as set during compile time of Node.js.
+
+Currently possible values are:
+
+* `'aix'`
+* `'darwin'`
+* `'freebsd'`
+* `'linux'`
+* `'openbsd'`
+* `'sunos'`
+* `'win32'`
+
+Equivalent to [`process.platform`][].
+
+*Note*: The value `'android'` may also be returned if the Node.js is built on
+the Android operating system. However, Android support in Node.js is considered
+to be experimental at this time.
 
 ## os.release()
 <!-- YAML
 added: v0.3.3
 -->
 
-Returns the operating system release.
+The `os.release()` method returns a string identifying the operating system
+release.
```

Comment:
It might be beneficial to add them here in the meantime. Especially for this function since it may not be obvious whether it returns an integer/double since it's typically a version number or if it's a string (which it is).

Additionally, there should be a note that this comes from `uname()` on *nix (and if you want to also mention the implementation for Windows, it uses `GetVersionExW()`).